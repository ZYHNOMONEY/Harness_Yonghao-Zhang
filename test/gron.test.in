{
    "company": "Tech Team",
    "employees": [
        {"name": "Yonghao", "department": "Engineering", "years": 10},
        {"name": "Zhang", "department": "Marketing", "years": 1}
    ],
    "location": {
        "city": "New York",
        "address": "233 Tech Street",
        "zip": "23333"
    },
    "revenue": 3222222,
    "publicly_traded": false
}
