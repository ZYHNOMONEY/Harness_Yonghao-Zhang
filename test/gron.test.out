json.company = "Tech Team"
json.employees.0.name = "Yonghao"
json.employees.0.department = "Engineering"
json.employees.0.years = 10
json.employees.1.name = "Zhang"
json.employees.1.department = "Marketing"
json.employees.1.years = 1
json.location.city = "New York"
json.location.address = "233 Tech Street"
json.location.zip = "23333"
json.revenue = 3222222
json.publicly_traded = false